licenses/antlr4-runtime-4.5.1-1.jar.sha1

```diff
@@ -0,0 +1 @@
+66144204f9d6d7d3f3f775622c2dd7e9bd511d97
```

Comment:
Why do we store the SHA1 of the licenses? It's probably obvious... but slow brain today...